File: wrappers/java/spring-demos/src/main/java/com/kendoui/spring/models/AprilSales.java
package com.kendoui.spring.models;

public class AprilSales {
    private int current;
    private int target;
    private String category;
    
    public AprilSales(int current, int target, String category) {
        setCurrent(current);
        setTarget(target);
        setCategory(category);
    }
    
    public int getCurrent() {
        return current;
    }
    
    public void setCurrent(int current) {
        this.current = current;
    }
    
    public int getTarget() {
        return target;
    }
    
    public void setTarget(int target) {
        this.target = target;
    }
    
    public String getCategory() {
        return category;
    }
    
    public void setCategory(String category) {
        this.category = category;
    }
}

File: wrappers/mvc/demos/Kendo.Mvc.Examples/Models/Chart/AprilSales.cs
namespace Kendo.Mvc.Examples.Models
{
    public class AprilSales
    {
        public AprilSales()
        {
        }

        public AprilSales(int current, int target, string category)
        {
            Current = current;
            Target = target;
            Category = category;
        }

        public string Category { get; set; }
        public int Current { get; set; }
        public int Target { get; set; }
    }
}